Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/GalleryMediaContent.java

```diff
@@ -203,8 +240,10 @@ private StreamedContent sendData(URI uri, String mimeType) {
          * that after transferring the data.
          */
         try {
-            InputStream previewData = ServiceManager.getFileService().read(uri);
-            return DefaultStreamedContent.builder().stream(() -> previewData).contentType(mimeType).build();
+            InputStream viewData = ServiceManager.getFileService().read(uri);
```

Comment:
PrimeResourceHandler should close the input stream.

https://searchcode.com/file/2686530/src/main/java/org/primefaces/application/PrimeResourceHandler.java/

Reply:
Sorry may comment was not right and long time since i implemented the RangeStreamContentHandler. I have adapted the primeresourcehandler like this. The input stream is also closed in the RangeStreamContentHandler. https://github.com/kitodo/kitodo-production/blob/39337a5949e5d03e888f67a6368f11ef0419424e/Kitodo/src/main/java/org/kitodo/production/handler/RangeStreamedContentHandler.java#L90

Nevertheless, I fit the try block.

Reply:
The following line `InputStream viewData = ServiceManager.getFileService().read(uri)` in try brackets, does not work for me. The video returns a server error ("input stream already closed") and is not rendered in the HTML video tag. Probably the stream is closed too early, so the `RangeStreamContentHandler` can't handle this stream. So I think that closing the stream only in the `RangeStreamContentHandler`, as described in the comment above, will be sufficient.